build-scripts/style-dictionary/config.js

```diff
@@ -53,6 +57,18 @@ module.exports = require('style-dictionary')
       bldPaths.srcTokensPath('**', '*.json')
     ],
     platforms: {
+      swift : {
+        transformGroup: 'ios-swift',
+        buildPath: `${bldApi.distPath('swift')}${path.sep}`,
+        files: [
+          {
+            className: 'StyleDictionaryColor',
```

Comment:
I think `GravityColors` would be a more appropriate name. "Style Dictionary" is just the tool we use to export our tokens in various formats, but as far as a consumer is concerned "Gravity colors" are what we're providing them with.

Reply:
agree, it is more meaningful.
Resolving as it is done if you don't mind @james-nash